exhibitor/migrations/0003_user_is_active.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.1 on 2023-02-08 20:00
```

Comment:
it should be one migration per PR for the same thing. What you can do now is:
- remove your all migrations
- revert change on models
- remove db and recreate it
- apply change on models
- create a new migration
- update repository